test/unit/jest.config.js

```diff
@@ -19,6 +19,7 @@ module.exports = {
 	],
 	testURL: 'http://localhost',
 	testPathIgnorePatterns: [
+		'/.git/',
```

Comment:
If these are regular expressions like the line three below would indicate, the `.` should be escaped as otherwise it would match any character, e.g. a folder `/agit/`

https://github.com/WordPress/gutenberg/blob/9d6a945d955d2cc8480ac9abe76ccbcecc3198d8/test/unit/jest.config.js#L25

```suggestion
		'/\.git/',
```